app/cdap/components/WrangleHome/Components/OngoingDataExploration/utils.ts

```diff
@@ -39,7 +39,7 @@ export const generateDataForExplorationCard = (oldData) => {
           obj.label = `${eachItem[keys]} Recipe steps`;
           obj.type = 'text';
         } else if (keys === 'dataQuality') {
-          obj.label = parseInt(eachItem[keys]);
+          obj.label = parseInt(eachItem[keys], 10);
```

Comment:
Please use `Number()` to parse numbers in base 10, per https://google.github.io/styleguide/tsguide.html#type-coercion

Reply:
Done.